src/Renderer/RasterTile.js

```diff
@@ -166,14 +166,22 @@ export class RasterElevationTile extends RasterTile {
     }
 
     initFromParent(parent, extents) {
+        const currentLevel = this.level;
         super.initFromParent(parent, extents);
         this.updateMinMaxElevation();
+        if (currentLevel !== this.level) {
+            this.dispatchEvent({ type: 'rasterElevationChanged', node: this });
+        }
     }
 
     setTextures(textures, offsetScales) {
+        const currentLevel = this.level;
         this.replaceNoDataValueFromTexture(textures[0]);
         super.setTextures(textures, offsetScales);
         this.updateMinMaxElevation();
+        if (currentLevel !== this.level) {
+            this.dispatchEvent({ type: 'rasterElevationChanged', node: this });
```

Comment:
Rename to `rasterElevationLevelChanged` to be more specific?

Reply:
yes